cachi2/interface/cli.py

```diff
@@ -102,6 +102,7 @@ def fetch_deps(
         ...,  # Ellipsis makes this option required
         help="Specify package (within the source repo) to process. See usage examples.",
         metavar="PKG",
+        callback=lambda args: [_if_json_then_validate(arg) for arg in args],
```

Comment:
Note: if the validation error happens in the callback, our error handler will **not** log it. This is true for all default Typer errors, such as non-existent parameters, missing `--source` directory etc.

How concerned are we about that? (Personally, not that much)

Reply:
I looks like all other Typer errors, I'd say it's fine.